apps/commune-governance/src/app/components/create-comment.tsx

```diff
@@ -12,7 +12,7 @@ import { api } from "~/trpc/react";
 
 const MAX_CHARACTERS = 300;
 const MAX_NAME_CHARACTERS = 300;
-const MIN_STAKE_REQUIRED = 5000;
+const MIN_STAKE_REQUIRED = 10;
```

Comment:
This value needs to be returned to 5000
```suggestion
const MIN_STAKE_REQUIRED = 5000;
```